lesson4/recursion2.js

```diff
@@ -0,0 +1,13 @@
+//create a createString function that takes (5,'hi') -> 'hihihihihi'
+
+let c = '';
```

Comment:
no global variable please.

Reply:
done. thanks! let me know if it needs more improvement